### PR TITLE
NFL test page

### PR DESCRIPTION
**Pull request by ibnulekram**
1. Open the web browser using WebDriver class. 
2. And did Assertion for validating the browser is now on a page titled "NFL Scoreboard".
3. I have used junit framework and for executing the test script I used Eclipse.

### CODE REVIEW

File: Ibnul Ekram [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import org.openqa.selenium.firefox.FirefoxDriver;
+public class NFLTest
+
+{
+@Test
+public void TestTitle()
+
+{
+WebDriver dr= new FirefoxDriver();
+dr.get("http://scores.espn.go.com/nfl/scoreboard");
```
Comment (jaxzin) [inline Ibnul Ekram]: Can you update this test to start on espn.go.com and click on the link for the NFL scoreboard?

Comment (ibnulekram) [inline Ibnul Ekram]: Hi there,
I have updated it. Here is the updated script

import junit.framework.Assert;

import org.junit.Test;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

public class NFLSCORE

{
@Test
public void TestTitle()

{
WebDriver dr= new FirefoxDriver();
dr.get("http://scores.espn.go.com/nfl/scoreboard");
        //Clicking on NFL link
dr.findElement(By.name("&lpos=sitenavdefault&lid=sitenav_nfl")).click();
        //Clicking on Score link
dr.findElement(By.name("&lpos=subnav&lid=subnav_NFL_Scoreboard")).click();

String ptitle=dr.getTitle();
Assert.assertTrue("Title not found!", ptitle.contains("NFL Scoreboard"));

}
}


Regards,

Ibnul Ekram
25 49 82 ST., East Elmhurst, NY 11370
Email: ibnulekram@gmail.com
Phone: 9292356101

Comment (ibnulekram) [inline Ibnul Ekram]: Hi there,
I have updated it. Here is the updated script


import junit.framework.Assert;
import org.junit.Test;
import org.openqa.selenium.By <http://org.openqa.selenium.by/>;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

public class NFLSCORE

{
@Test
public void TestTitle()

{
WebDriver dr= new FirefoxDriver();
dr.get("espn.go.com");
        //Clicking on NFL link
dr.findElement(By.name("&lpos=sitenavdefault&lid=sitenav_nfl")).click();
        //Clicking on Score link
dr.findElement(By.name("&lpos=subnav&lid=subnav_NFL_Scoreboard")).click();

String ptitle=dr.getTitle();
Assert.assertTrue("Title not found!", ptitle.contains("NFL Scoreboard"));

}
}

Regards,

Ibnul Ekram
25 49 82 ST., East Elmhurst, NY 11370
Email: ibnulekram@gmail.com
Phone: 9292356101

Comment (jaxzin) [inline Ibnul Ekram]: Can you commit the change to git and repush your `patch-1` branch?

**Pull request closed without merging**